docs/community/03_governance_overview.md

```diff
@@ -34,8 +42,9 @@ Radworks’ treasury is controlled by an on-chain governance mechanism ([Compoun
   </Button>
 </Grid>
 
-### Contracts
+### Relevant Contracts
 
+* [Radworks Governance Contracts](https://github.com/radicle-foundation/radworks-governance/blob/main/manual.md#formal-review)
 * Governance: [0x690e775361AD66D1c4A25d89da9fCd639F5198eD](https://etherscan.io/address/0x690e775361AD66D1c4A25d89da9fCd639F5198eD)
```

Comment:
Do we want to clarify what this actually linking to? Maybe "Governance Record"?

Reply:
No, lets just leave it as is